sdl-core/src/main/scala/io/smartdatalake/workflow/connection/jdbc/JdbcTableConnection.scala

```diff
@@ -50,11 +51,9 @@ import java.sql.{DriverManager, ResultSet, Statement, Connection => SqlConnectio
  *                               Note that Spark manages JDBC Connections on its own. This setting only applies to JDBC connection
  *                               used by SDL for validating metadata or pre/postSQL.
  * @param connectionPoolMaxIdleTimeSec timeout to close unused connections in the pool
- * @param enableCommit whether to enable commits for transactional behaviour in SDL
- *                     Note that most JDBC drivers use auto-commit per default which is not recommended when [[enableCommit]] is true.
- *                     Default is enableCommit = true. To disable auto-commit see [[autoCommit]].
- * @param autoCommit flag to enable or disable the auto-commit behaviour of the JDBC driver
- *                   If not set, the default auto-commit mode of the JDBC driver is used.
+ * @param connectionPoolMaxWaitTimeSec timeout when waiting for connection in pool to become available. Default is 60 seconds.
```

Comment:
I added `connectionPoolMaxWaitTimeSec` because otherwise it is hard to detect that the application hangs because it waits for a connection and that `maxParallelConnectinos` should be increased. In addition, if due to an error in the code the connection is not returned to the pool correctly, the application might hang indefinitely without logging why. We can increase the default to 10m because waiting a few minutes more or less is usually not a problem for SDLB.